cpp/include/IceUtil/Exception.h

```diff
@@ -21,16 +21,16 @@ namespace IceUtil
     {
     public:
         /**
-         * Constructs the exception. Equivalent to Exception(nullptr, 0).
+         * Default constructor. Equivalent to Exception(nullptr, 0).
          */
-        Exception();
+        Exception() noexcept;
```

Comment:
Do we ever use this one?

Reply:
Yes, we use the default constructor quite a bit. For example, user exceptions usually do use this ctor.